src/plugins/intel_gna/backend/gna_limitations.cpp

```diff
@@ -118,55 +118,113 @@ std::string RectLimitByChannelsAndPrecision::GetErrorOrEmpty(const uint32_t h, c
+const RangeLimit Validator_35::kInputChannelsNumberLimit1B{1, 2048, "number of input channels"};
+const RangeLimit Validator_35::kInputChannelsNumberLimit2B{1, 1024, "number of input channels"};
+
+const RangeLimit Validator_35::kKernelNumberLimit{1, 8192, "number of kernels"};
+const RangeLimit2D Validator_35::kKerneHWlLimit{{1, 255, "kernel height"}, {1, 256, "kernel width"}};
+const RangeLimit2D Validator_35::kStrideHWLimit{{1, 255, "convolution stride height"},
+                                                {1, 256, "convolution stride width"}};
+const RangeLimit2D Validator_35::kDilationLimit{{convDilationHeight, convDilationHeight, "dilation height"},
+                                                {convDilationWidth, convDilationWidth, "dilation width"}};
+
+bool Validator_35::ValidateCnn2D(const std::string& name, const uint32_t inHeight, const uint32_t inWidth,
+    const uint32_t inChannels, const uint32_t kernelH, const uint32_t kernelW, const uint32_t kernelN,
+    const uint32_t strideH, const uint32_t strideW, const uint32_t dilationH, const uint32_t dilationW,
+    const OvGnaType inPrecision, const bool throwOnError) const {
+    auto error = kInputHWLimit.GetErrorOrEmpty(inHeight, inWidth);
+
+    error += kKernelNumberLimit.GetErrorOrEmpty(kernelN);
+    auto& inputChannelsNumberLimit = inPrecision == OvGnaTypeInt8 ? kInputChannelsNumberLimit1B : kInputChannelsNumberLimit2B;
+    error += inputChannelsNumberLimit.GetErrorOrEmpty(inChannels);
+    error += kKerneHWlLimit.GetErrorOrEmpty(kernelH, kernelW);
+    error += kStrideHWLimit.GetErrorOrEmpty(strideH, strideW);
+
+    error += kDilationLimit.GetErrorOrEmpty(dilationH, dilationW);
+
+    return ValidationSuccesful(throwOnError, error, name, "Convolution2D");
+}
+
+const RangeLimit2D Validator_35::kPoolingWindowHWLimit{{1, 255, "pooling window height"},
+                                                       {1, 255, "pooling window width"}};
+const RangeLimit2D Validator_35::kPoolingStrideHWLimit{{1, 255, "pooling stride height"},
+                                                       {1, 255, "pooling stride width"}};
+
+bool Validator_35::ValidatePooling2D(const std::string& name,
+    const uint32_t windowH, const uint32_t windowW,
+    const uint32_t strideH, const uint32_t strideW,
+    const bool throwOnError) const {
+    auto error = kPoolingWindowHWLimit.GetErrorOrEmpty(windowH, windowW);
+    error += kPoolingStrideHWLimit.GetErrorOrEmpty(strideH, strideW);
 
-    return error.empty() ? true : false;
+    return ValidationSuccesful(throwOnError, error, name, "Pooling2D");
+}
+
+bool Validator_35::IsPaddingSupported() const {
+    return true;
 }
 
 std::unique_ptr<AbstractValidator> AbstractValidator::Create(const std::string& target) {
     if (target == InferenceEngine::GNAConfigParams::GNA_TARGET_3_0) {
         return tools::make_unique<Validator_30>();
+    } else if (target == "GNA_TARGET_3_5") {
```

Comment:
Shouldn't we add 
DECLARE_GNA_CONFIG_VALUE(TARGET_3_5);
in gna_config.hpp

Reply:
see
https://github.com/openvinotoolkit/openvino/pull/11307#discussion_r838215111